doc/tomb.1

```diff
@@ -424,12 +424,23 @@ To avoid that tomb execution is logged by \fIsyslog\fR also add:
 
 Password input is handled by the pinentry program: it can be text
 based or graphical and is usually configured with a symlink. When
-using Tomb in X11 it is better to use a graphical pinentry-gtk2 or
-pinentry-qt because it helps preventing keylogging by other X
-clients. When using it from a remote ssh connection it might be
-necessary to force use of pinentry-curses for instance by unsetting
-the DISPLAY environment var.
+using Tomb in a graphical environment (X11 or Wayland) it is better
+to use either pinentry-gtk2 (deprecated), pinentry-gnome or
+pinentry-qt because it helps preventing keylogging by other clients.
+When using it from a remote ssh connection it might be necessary to
+force use of pinentry-curses for instance by unsetting the DISPLAY
```

Comment:
or pinentry-tty, actually even better in these situations.

Reply:
Okay, will change that